### PR TITLE
Add a directory & best practices document for working groups

### PR DESCRIPTION
**Pull request by mrjoro**
This adds a directory to the meta repo for documentation related to Working Groups and a document with Working Group best practices (what the WG should document, where discussions should happen, etc.). 

It also adds a README.md that we can use for listing the Working Groups once the TSC creates them.

Sending to @aghassemi for review, but anyone in the community is welcome to provide input here.

/cc @cramforce @tobie @adelinamart

### CODE REVIEW

File: working-groups/best-practices.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+This document describes best practices for the operation of AMP Working Groups.  Working Groups may deviate from these best practices in reasonable ways that work best for the Working Group as long as those changes align with [AMP’s governance policy](../GOVERNANCE.md).
+
+* GitHub repository.  Create an ampproject/wg-{name} GitHub repository (e.g. “ampproject/wg-ui”).
+  * Link to this repo from the [ampproject/meta/working-groups/README.md](README.md) file.
+  * Use this repo to keep for documentation about the working group, meeting agendas, etc.  **This repo should not contain code.**
+  * This repository should contain a README.md with enough information to make it possible to keep track of all work that is happening within the WG and all of the decisions that require the attention of WG members.  This should include:
+    * the WG’s mandate from the TSC (e.g. which repositories and parts of the codebase the WG is responsible for)
+    * a list of members
+    * how the working group operates, including discussion channels, how decisions are made, and how issue labels/GitHub projects are used.  This documentation should.
```
Comment (aghassemi) [inline working-groups/best-practices.md]: typo: extra `This documentation should.` at the end of the sentence.

**Comment by mrjoro** [comment]
@tobie cool!  Currently for repos we need to go through a separate process (that's mostly boilerplate); if/when AMP moves to a foundation we should do the repo creation script.  I considered a template for the README, but I think it'll be easier to just copy an existing README and modify it--since then the sections would be self explanatory.

@cramforce I'll send a follow-up PR mandating that all documentation have two spaces after the period.  Or should we add this to GOVERNANCE.md? ;)

File: working-groups/best-practices.md (hunk 1 above)
Comment (mrjoro) [reply working-groups/best-practices.md]: Done!

**Comment by tobie** [comment]
> I'll send a follow-up PR mandating that all documentation have two spaces after the period. Or should we add this to GOVERNANCE.md? ;)

@cramforce's comment cracked-me up. I've been meaning to bring that up since forever. :)

**Pull request merged**